basculechecks/capabilitiesvalidator.go

```diff
@@ -135,9 +136,9 @@ func getCapabilities(attributes bascule.Attributes) ([]string, string, error) {
 		return []string{}, UndeterminedCapabilities, fmt.Errorf("couldn't get capabilities using key %v", CapabilityKey)
 	}
 
-	vals, ok := val.([]string)
-	if !ok {
-		return []string{}, UndeterminedCapabilities, fmt.Errorf("capabilities value not the expected string slice: %v", val)
+	vals, err := cast.ToStringSliceE(val)
```

Comment:
I've run into this many times as well... The val is actually of type `[]interface{}` if I remember correctly.